src/status_im/chat/views/message.cljs

```diff
@@ -169,11 +169,11 @@
 
 (defn get-style [string]
   (->> replacements
-       (into [] (comp
-                  (map first)
-                  (map #(vector % (re-pattern %)))
-                  (drop-while (fn [[_ regx]] (not (re-matches regx string))))
-                  (take 1)))
+       (vec (comp
```

Comment:
This is wrong, looks like that `kibit` "doesn't know" about the version of `into` that was introduced along with transducers.
https://github.com/clojure/clojurescript/blob/master/src/main/cljs/cljs/core.cljs#L4788